### PR TITLE
feat(superset-ui-connection): Allow PUT and PATCH in call API

### PR DESCRIPTION
**Pull request by fabiomendescom**
🏆 Enhancements

Feat: Support `PUT` and `PATCH`. Address #104

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -94,5 +94,8 @@
       "yarn run prettier --write",
       "git add"
     ]
+  },
```
Comment (kristw) [inline package.json]: Is this intentional?

Comment (williaster) [reply package.json]: yeah shouldn't be there in a `package.json`. if it's auto-formatting you could call `npm run lint` from the monorepo root (which should generate eslint + prettier config) and it hopefully will stop that.

Comment (williaster) [reply package.json]: yeah this should be installed via the `travis.yml`?

**Comment by kristw** [comment]
@fabiomendescom Your changes look good. Please address the `codecov` issue above and we can merge.

**Comment by fabiomendescom** [comment]
> @fabiomendescom Your changes look good. Please address the `codecov` issue above and we can merge.

I'm not sure what I need to address. It says 100% coverage. What exactly do I need to do?

File: package.json (hunk 1 above)
Comment (fabiomendescom) [reply package.json]: Im a bit confused about this. This file was not modified in this PR. What do I need to do?

Comment (williaster) [reply package.json]: just remove this added dependency, it seems to have found it's way into the commit/PR even if you didn't intend it to?

**Pull request merged**